Modules/Devices/src/Devices.Application/PushNotifications/Commands/DeleteRegistrationsByIdentityAddress/Handler.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using MediatR;
+
+namespace Backbone.Modules.Devices.Application.PushNotifications.Commands.DeleteRegistrationsByIdentityAddress;
+public class Handler(IPnsRegistrationsRepository pnsRegistrationRepository) : IRequestHandler<DeleteRegistrationsByIdentityAddressCommand>
+{
+    public async Task Handle(DeleteRegistrationsByIdentityAddressCommand request, CancellationToken cancellationToken)
```

Comment:
Should we have a validator for the command payload?

Reply:
I have added a validator for all Commands extending `RequestWithIdentityAddress`, though I'm still unsure if it's being used. I need to test further.

Reply:
It's not ideal to introduce the extra complexity of inheritance just to save a property. Adding the property is a one-time thing. And even the validator you're probably trying to avoid isn't that hard to implement.

I understand your urge to get rid of this duplication. I've made the same mistake (using inheritance wherever possible) in the past. But after reading a few books on the subject, I realised that it might not be the best idea.

Reply:
In this case, the usage of inheritance allows us (I think) to eliminate the redundancy of Validators, i.e., we only need an implementation of a Validator for all Commands. Isn't that reason enough to keep it? Or this there another way to have such a Validator without inheritance?

Reply:
This is not how the validators work. You can see how the validation is called in https://github.com/nmshd/backbone/blob/update-deletions-process/BuildingBlocks/src/BuildingBlocks.Application/MediatR/RequestValidationBehavior.cs

Reply:
@tnotheis I have removed the interface and added a validator for each command. See:
 - 220e406a2d5ea0864c614c09499c6d54fdc0e561
 - ebab6bb04c1d8edc3b2cffc43344f215be1741d8